_posts/2021-02-09-january-rubygems-update.md

```diff
@@ -0,0 +1,37 @@
+---
+title: January 2021 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in January.
+
+## RubyGems News
+
+In January, we released new versions of bundler `v2.2.7` and `v2.2.8`, and rubygems `v3.2.7` and `v3.2.8`, the most important change being to fix resolver edge cases and speed up bundler - [#4277](https://github.com/rubygems/rubygems/pull/4277) this change makes bundler faster and the resolver more correct.
+
+We also made the following improvements and fixes:
+
+- looked up information for a CI related issue where the the `--silent` flag did not work with `gem update --system` - [#2907](https://github.com/rubygems/rubygems/issues/2907#issuecomment-756957235) and that was resolved on [#3789](https://github.com/rubygems/rubygems/pull/3789)
```

Comment:
This bullet point doesn't seem important enough to me, specially since it didn't lead to any changes in rubygems. I'd unify this kind of change under a generic entry, like. "As always, we keep triaging new issues and reviewing pull requests from contributors".

Reply:
Alright noted will update this.